src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -91,30 +98,24 @@
     margin-top: 0.5rem;
     width: 90%;
     max-width: 800px;
-  }
+    background-color: transparent;
+    padding: $focus-ring-width;
+    border: none;
 
-  .search-button__overlay {
-    display: flex;
-    position: fixed;
-    z-index: $zindex-modal-backdrop;
-    background-color: black;
-    opacity: 0.5;
-    width: 100%;
-    height: 100%;
-    top: 0;
-    left: 0;
-  }
+    &::backdrop {
+      background-color: black;
+      opacity: 0.5;
+    }
 
-  form.bd-search {
-    flex-grow: 1;
-    padding-top: 0;
-    padding-bottom: 0;
```

Comment:
These padding rules are not needed. The form already has 0 top and bottom padding.